app/components/Tags/index.tsx

```diff
@@ -10,7 +11,15 @@ type Props = {
  * A basic tag component for displaying tags
  */
 const Tags = ({ children, className }: Props) => {
-  return <div className={className ? className : styles.tags}>{children}</div>;
+  return (
+    <Flex
+      wrap
+      gap="var(--spacing-sm)"
+      className={className ? className : styles.tags}
```

Comment:
I would think that `styles.tags` doesn't exist any more
```suggestion
      className={className}
```